pylint_odoo/test_repo/broken_module2/tests/data/odoo_data_noupdate_1.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+    </data>
+    <data>
+    </data>
+</odoo>
```

Comment:
Could you please add a test case with the [example](https://github.com/OCA/pylint-odoo/pull/129#issuecomment-306834319) provided by Stéphane?

```xml
<odoo>
  <data noupdate="0">
  </data>
  <data noupdate="1">
  </data>
</odoo>
```